content/security/cors.md

```diff
@@ -20,3 +20,18 @@ Alternatively, enable CORS via the `create()` method's options object. Set the `
 const app = await NestFactory.create(AppModule, { cors: true });
 await app.listen(3000);
 ```
+
+#### GraphQL
```

Comment:
I would delete the title, otherwise it should also be placed above for REST endpoints, instead without I find it better.

Reply:
So just delete the GraphQL title?

Reply:
In my opinion untitled could also go, it links well to the title above `Getting started`.

@jmcdo29  what do you think about it?

Reply:
I agree here, remove the GraphQL title and let it be a single page. Should be fine. If we had more than just these two titles _may_ be useful, but with just two I think this is fine.

Reply:
I've fixed it